docker-run

```diff
@@ -96,7 +96,6 @@ def main():
     else:
         # For Jenkins slave in Docker
         docker_adrs = netifaces.gateways()[netifaces.AF_INET][0][0]
-    docker_cmd.append("--env=HOME_DIR={}".format(os.environ["HOME"]))
```

Comment:
HOME at line 86 should be changed to `HOME_DIR`...

Reply:
I do not understand why we need a HOME_DIR variable, all times equals to HOME.

Reply:
This cause a lot of confusion as we set HOME=HOME_DIR and HOME_DIR=HOME so where is the chicken and where is the egg ?

Reply:
Yes but at line 86 https://github.com/camptocamp/c2cgeoportal/blob/17507da584407ab6e5d08fa90c72ab1d5aa3b8de/docker-run#L86 there still have a deference on ``HOME``, that should be changed to ``HOME_DIR``, to have ``--env=HOME_DIR`` in the command line to transmit the ``HOME_DIR`` ...

Reply:
Sorry I'm wrong I switch HOME and DOME_DIN in my head...